app/bundles/LeadBundle/Views/Company/company.html.php

```diff
@@ -0,0 +1,293 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
```

Comment:
All new files should be marked with actual year for copyright (2020)

Reply:
I changed the year!